interface/usergroup/addrbook_edit.php

```diff
@@ -29,6 +29,7 @@
  $info_msg = "";
 
  function invalue($name) {
+  if (!$_POST[$name]) return "''";
   $fld = add_escape_custom(trim($_POST[$name]));
   return "'$fld'";
```

Comment:
I have an update to add suffix to the address book.
Letters look funny without the suffix and the address book looks funny with fname: John lmane: Smith, MD.  The suffix needs a home...